pkg/api/aim/dashboards.go

```diff
@@ -151,18 +169,28 @@ func UpdateDashboard(c *fiber.Ctx) error {
 		},
 	}
 	if err := database.DB.
-		Joins("LEFT JOIN apps ON dashboards.app_id = apps.id").
+		InnerJoins(
+			"App",
+			database.DB.Select(
+				"NamespaceID",
```

Comment:
I think we would be better off selecting `ID` and `Type` instead, just so we are extra safe in _not_ updating the app further. We already use `Omit("App")` to avoid it, and with this we'll be even safer.

Reply:
ook!